docs/api-reference/next.config.js/rewrites.md

```diff
@@ -4,6 +4,13 @@ description: Add rewrites to your Next.js app.
 
 # Rewrites
 
+<details>
+  <summary><b>Examples</b></summary>
+  <ul>
+    <li><a href="https://github.com/vercel/next.js/tree/canary/examples/rewrites">Rewrites - Custom Routes</a></li>
```

Comment:
Rewrites is enough, that's the feature is called, we don't mention custom routes anywhere.

```suggestion
    <li><a href="https://github.com/vercel/next.js/tree/canary/examples/rewrites">Rewrites - Custom Routes</a></li>
```